src/realm/object-store/sync/sync_session.cpp

```diff
@@ -522,16 +560,6 @@ void SyncSession::handle_progress_update(uint64_t downloaded, uint64_t downloada
     m_progress_notifier.update(downloaded, downloadable, uploaded, uploadable, download_version, snapshot_version);
 }
 
-void SyncSession::create_sync_session()
```

Comment:
This just seemed pointless? I assume the `create_sync_session()`/`do_create_sync_session()` split used to make sense, but it doesn't now.